packages/client/src/components/blocks-workspace/BlocksRenderer.tsx

```diff
@@ -131,6 +148,7 @@ export const BlocksRenderer = observer((props: BlocksRendererProps) => {
                             'Please be mindful this may not represent the current state of the app, due to the filters present in the URL',
                     });
                 }
+                findBlockFromRoute(s.blocks);
```

Comment:
can we remove this here with the lower useEffect?

Reply:
updated